cfn-resources/cloud-provider-access/README.md

```diff
@@ -0,0 +1,13 @@
+# Mongodb::Atlas::CloudProviderAccess
+
+Congratulations on starting development!
```

Comment:
Can you copy format from other resources for README file? This looks off...

Reply:
@parthasarathy-varadhan you had a readme file you told me you want to add?